### ISSUE TITLE
Changes in `syn_registration_3d` example

### ISSUE THREAD

**Issue by arokem** [nipy/dipy#701]
With current master, I am getting: 

```
(py2)D-69-91-145-243:examples (master) $python syn_registration_3d.py 
Dataset is already in place. If you want to fetch it again please first remove the folder /Users/arokem/.dipy/stanford_hardi 
Dataset is already in place. If you want to fetch it again please first remove the folder /Users/arokem/.dipy/syn_test 
Traceback (most recent call last):
  File "syn_registration_3d.py", line 73, in <module>
    resampled = vfu.warp_3d_affine(moving.astype(np.float32), 
AttributeError: 'module' object has no attribute 'warp_3d_affine'
```

**Comment by omarocegueda** [comment]
Hi @arokem,
thanks for reporting this! 
Here is the proposed fix: #703

**Comment by arokem** [comment]
This was closed through #703
